PostFreeSurfer/scripts/FreeSurfer2CaretConvertAndRegisterNonlinear.sh

```diff
@@ -213,6 +213,12 @@ for GrayordinatesResolution in ${GrayordinatesResolutions} ; do
     ${CARET7DIR}/wb_command -volume-label-import "$AtlasSpaceFolder"/ROIs/wmparc."$GrayordinatesResolution".nii.gz ${SubcorticalGrayLabels} "$AtlasSpaceFolder"/ROIs/ROIs."$GrayordinatesResolution".nii.gz -discard-others
     [ "${T2wImage}" != "NONE" ] && applywarp --interp=spline -i "$AtlasSpaceFolder"/"$AtlasSpaceT2wImage".nii.gz -r "$AtlasSpaceFolder"/ROIs/Atlas_ROIs."$GrayordinatesResolution".nii.gz -o "$AtlasSpaceFolder"/"$AtlasSpaceT2wImage"."$GrayordinatesResolution".nii.gz
     applywarp --interp=spline -i "$AtlasSpaceFolder"/"$AtlasSpaceT1wImage".nii.gz -r "$AtlasSpaceFolder"/ROIs/Atlas_ROIs."$GrayordinatesResolution".nii.gz -o "$AtlasSpaceFolder"/"$AtlasSpaceT1wImage"."$GrayordinatesResolution".nii.gz
+    applywarp --interp=nn -i "$AtlasSpaceFolder"/brainmask_fs.nii.gz -r "$AtlasSpaceFolder"/ROIs/Atlas_ROIs."$GrayordinatesResolution".nii.gz -o "$AtlasSpaceFolder"/brainmask_fs."$GrayordinatesResolution".nii.gz
```

Comment:
Opps.  Must not have saved my comment on this line, which was to make use of the defined variable `$T1wImageBrainMask` rather than relying on the hardcoded file name `brainmask_fs`.

Reply:
OK